releng/setup-env.sh

```diff
@@ -466,7 +466,7 @@ case $host_platform in
     OBJCOPY="${android_toolroot}/bin/${host_toolprefix}objcopy"
     OBJDUMP="${android_toolroot}/bin/${host_toolprefix}objdump"
 
-    CFLAGS="$host_arch_flags -DANDROID -fPIE -ffunction-sections -fdata-sections"
+    CFLAGS="$host_arch_flags -DANDROID -fPIC -ffunction-sections -fdata-sections"
```

Comment:
Oops. I think I clicked the resolve button on accident.  Sorry about that.  I'll submit another PR for the suggested change.